### ISSUE TITLE
muted option or velocity for each note

### ISSUE THREAD

**Issue by qcmv22** [trackbout/ripchord#5]
Will be usefull to muted some key... like usely scales or sieves always have less notes than the total chromatic.... But I realize might be difficult to program.... maybe by assigning velocity to each note then muted might be setting velocity to 0 .

Anyhow very good and usefull software.

**Comment by playskeyskc** [comment]
I agree with qcmv22 that first and foremost, this is an AWESOME plugin and I'm very glad you created it. I was thinking about the velocity per note in a chord from the perspective of being able to set each note to add or subtract a 'range' of velocity from each note. Along with your current humanize feature, this would go one step further to providing a more humanly played representation of a give chord. If this is something you'd actually entertain, I'd help put together a use case and requirements for it as well. Yes, I am in software development. If c++ was in my repertoire, I would try to write the enhancement myself and send you a pull request.
